### ISSUE TITLE
Setting verbose per feed in cfg is ignored

### ISSUE THREAD

**Issue by xrat** [rss2email/rss2email#190]
My default `verbose` setting is `verbose = warning` as set at the beginning of `rss2email.cfg`. I'd like to change the verbosity for some feeds but the `verbose` in

```
[feed.example] 
url = https://www.example.net/
verbose = error
```

is apparently ignored. I tried changing it to `debug` but it still does not produce debug output.

**Comment by Ekleog** [comment]
Right, this is currently not a supported use case: the `verbose` setting can only be global. To be honest, I'm not sure this feature would be worth the additional complexity of having to thread through logging levels for each feed.

Do you have an use case that could maybe help demonstrate the need?

**Comment by xrat** [comment]
My actual use case is that for some of my feeds I am (or feel) responsible and I want to see warnings because they tell me something is not as it should be (e.g. wrong encoding, or slow response). The majority of my other feeds does not produce warnings, or at least not frequently. So these do not matter. But some feeds generate warnings frequently which is more than annoying (it gets harder to see the important warnings when there are many others). So I wanted to silence them.

However, clearly, this is related to the bigger, currently ongoing discussion of r2e's logging verbosity, cf. #180, #187.

I opened this issue because I was not sure whether it was an oversight or not. Of course, logging levels for each feed would be nice, but I understand very well that coding complexity is to be considered. For my use case there are ways to solve it by other means (e.g. filtering r2e's output as in `r2e run | grep -Ev ...`). 

However, if per feed verbosity is not going to be added please consider adding this to the documentation.

**Comment by auouymous** [comment]
Have you tried `r2e -V run --clean` one day of the week (with or without the --clean), `r2e run` on the others, and `verbose = error` in the config?

**Comment by xrat** [comment]
@auouymous for the feeds where I want to see warnings I want to see them at the next scheduled run. What you suggested would on average delay these warnings.

**Comment by Ekleog** [comment]
@xrat To be honest I don't think I can imagine the implementation being simple enough to be reasonable to merge, when per-component verbosity isn't an expectation of the wider ecosystem and there are easy-ish workarounds. So I'm renaming this issue to reflect the lack of documentation, unless someone comes up with a simple enough implementation. Thank you for your report!

As for workarounds, have you tried having two r2e configuration files, one with your feeds and one for the feeds you're fetching from other people, and then running two r2e crons each with its own verbosity? It's a bit more infrastructure to setup but should hopefully allow you to get the exact same result as if there were per-feed verbosity levels :)

**Comment by Ekleog** [comment]
Just to check, would fixing the issue described by the new title match your documentation expectations? I'm thinking having the program itself tell you it's wrong would be better than having it just in the documentation, especially as having it in the documentation would mean adding for each key that it can't be used under a feed block :)

**Comment by auouymous** [comment]
I agree, having the program throw an error would be better. Although, there are only a few keys used by feeds and the documentation could mention the ones allowed in feeds.

**Comment by xrat** [comment]
> I don't think I can imagine the implementation being simple enough to be reasonable to merge

OK.

> So I'm renaming this issue to reflect the lack of documentation, unless someone comes up with a simple enough implementation. Thank you for your report!

Thank you!

> As for workarounds, have you tried having two r2e configuration files

Good idea, though it does feel a bit awkward to have 2 files with the same global config except for 1 line ;) Anyway, as you said, there are "easy-ish workarounds", multiple conf files being one.

> would fixing the issue described by the new title match your documentation expectations?

Sure thing, thanks.

> having the program throw an error would be better. Although, there are only a few keys used by feeds and the documentation could mention the ones allowed in feeds.

+1
